.changeset/dull-otters-smash.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/world-modules": patch
+---
+
+Updated the ERC20 and ERC721 implementations to always register the token namespace.
```

Comment:
```suggestion
ERC20 and ERC721 implementations now always register the token namespace, instead of checking if it has already been registered. This prevents issues with registering the namespace beforehand, namely that only the owner of a system can create a puppet for it.
```

Reply:
Thank you, changelog 👑